README.md

```diff
@@ -1,27 +1,58 @@
-# Frobnicator
+
+# Pierwsza iteracja
+
+## edytor tekstowy w ruście
+
+Pierwsza część projektu zawiera edytor tekstowy obsługujący:
+- formatowanie w stylu MarkDown'a, m.in: \*pogrubienie\*, \/kursywa/\, \`kod\`, \$indeks dolny\$, \^indeks górny\^, \~przekreślenie\~, \_podkreślenie\_, \<url\>
+- wyświetlanie sformatowanego tekstu na bieżąco w równoległym oknie (podgląd można wyłączyć)
+- skróty klawiszowe: ctrl + B: *strong*  ctrl + D: `code`  ctrl + I: /italics/  ctrl + L: $lowered$  ctrl + R: ^raised^  ctrl + S: ~strikethrough~  ctrl + U: _underline_  ctrl + Q: quit
+- obsługę plików: otwieranie pliku, zapis pliku, zapis pliku pod nową nazwą, zapis pliku przy wychodzeniu z edytora
+
+## uruchamianie
+
+1. `sudo apt-get install libxcb-render0-dev libxcb-shape0-dev libxcb-xfixes0-dev libspeechd-dev libxkbcommon-dev libssl-dev` (na linuxie)
```

Comment:
Po pierwsze to na Ubuntu, bo na innych dystrybucjach to mogą być inne pakiety, a po drugie musiałem jeszcze jakieś inne pakiety poinstalować (to drobnostka, ale gdybyście komuś innemu chcieli to pokazać to warto zmienić)